packages/wrangler/src/versions/index.ts

```diff
@@ -272,5 +276,11 @@ export default function registerVersionsSubcommands(
 			(yargs) => {
 				return registerVersionsSecretsSubcommands(yargs.command(subHelp));
 			}
+		)
+		.command(
+			"secret:bulk [json]",
+			"Create or update a secret variable for a Worker",
+			versionsSecretsPutBulkOptions,
+			versionsSecretPutBulkHandler
```

Comment:
just to be abs sure I'm understanding...`wrangler secret:bulk` and `wrangler versions secret:bulk` have exactly the same behaviour?